napalm/nxos_ssh/nxos_ssh.py

```diff
@@ -584,16 +585,23 @@ def get_facts(self):
         show_int_status = self._send_command("show interface status")
         show_hostname = self._send_command("show hostname")
 
+        show_inventory_table = self._get_command_table(
+            "show inventory | json", "TABLE_inv", "ROW_inv"
+        )
+        if isinstance(show_inventory_table, dict):
+            show_inventory_table = [show_inventory_table]
+
+        for row in show_inventory_table:
+            if row["name"] == "Chassis":
+                serial_number = row.get("serialnum", "")
```

Comment:
Similar comment as above (in `nxos.py`).

Reply:
Already defined in line 580